paddle/fluid/operators/range_op.cc

```diff
@@ -67,6 +67,21 @@ class RangeOp : public framework::OperatorWithKernel {
     ctx->SetOutputDim("Out", {-1});
   }
 
+  framework::OpKernelType GetExpectedKernelType(
+      const framework::ExecutionContext &ctx) const override {
+    auto input_data_type =
+        framework::OperatorWithKernel::IndicateVarDataType(ctx, "Start");
+
+#ifdef PADDLE_WITH_MKLDNN
+    if (this->CanMKLDNNBeUsed(ctx, input_data_type)) {
```

Comment:
Do you need to check AVX512 pre-condition here ? Could you actually paste relevant doc or manual for this kernel link here ? Thanks !